### ISSUE TITLE
test_simple.py 'KaldiRecognizer' object has no attribute 'SetPartialWords'

### ISSUE THREAD

**Issue by annesodub** [alphacep/vosk-api#935]
I just installed (clone from github) vosk on my Mac (iOS Catalina 10.15.3). The installation went well following the instructions. However, the execution of 

`python3 ./test_simple.py test.wav   
`
gives the following error :  

```
LOG (VoskAPI:ReadDataFiles():model.cc:213) Decoding params beam=10 max-active=3000 lattice-beam=2
LOG (VoskAPI:ReadDataFiles():model.cc:216) Silence phones 1:2:3:4:5:6:7:8:9:10
LOG (VoskAPI:RemoveOrphanNodes():nnet-nnet.cc:948) Removed 0 orphan nodes.
LOG (VoskAPI:RemoveOrphanComponents():nnet-nnet.cc:847) Removing 0 orphan components.
LOG (VoskAPI:CompileLooped():nnet-compile-looped.cc:345) Spent 0.058331 seconds in looped compilation.
LOG (VoskAPI:ReadDataFiles():model.cc:248) Loading i-vector extractor from model/ivector/final.ie
LOG (VoskAPI:ComputeDerivedVars():ivector-extractor.cc:183) Computing derived variables for iVector extractor
LOG (VoskAPI:ComputeDerivedVars():ivector-extractor.cc:204) Done.
LOG (VoskAPI:ReadDataFiles():model.cc:281) Loading HCL and G from model/graph/HCLr.fst model/graph/Gr.fst
LOG (VoskAPI:ReadDataFiles():model.cc:302) Loading winfo model/graph/phones/word_boundary.int
Traceback (most recent call last):

File "./test_simple.py", line 22, in <module>
    rec.SetPartialWords(True)
AttributeError: 'KaldiRecognizer' object has no attribute 'SetPartialWords'

```

Following [this recommendation](https://github.com/alphacep/vosk-api/issues/595), I tried 

`  pip3 install -U vosk`

with no success.

Would you please let me know if I am missing anything?

Thank you!

**Comment by nshmyrev** [comment]
Comment out that line

**Comment by Gordo851** [comment]
I had the same issue in windows. In Linux there's a more recent distribution where that method does exist in the KaldiRecognizer and it worked perfectly. If all you want is the final result though then yes, just comment out that line and it works.

**Comment by nshmyrev** [comment]
We released 0.3.38, it should work now
